Assets/Editor/CustomGui.cs

```diff
@@ -197,11 +198,22 @@ public void OnGUI()
             drawTree(currentTree);
         }
 
+        float biggestX = 0;
+        // Find the highest x position.
+        for (int i = 0; i < atLayer.Count; i++)
+        {
+            if (atLayer[i] > biggestX)
+                biggestX = atLayer[i];
+            
+        }
+
+        GUI.backgroundColor = Color.clear;
+        GUILayout.Box("", GUILayout.Height(-15), GUILayout.Width(biggestX * 200 + 20));
+        GUILayout.Box("", GUILayout.Height(atLayer.Count *100 + 100), GUILayout.Width(0));
```

Comment:
Should this be based on the screen restitution and not hard coded?

Reply:
screen resolution can only be obtained for the play screen, and not for Editor extensions.